docs/data-management-on-carbondata.md

```diff
@@ -333,6 +373,20 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
      ```
      ALTER TABLE test_db.carbon CHANGE a1 a1 DECIMAL(18,2)
      ```
+   - **SET and UNSET for Local Dictionary Properties**
+   
+      When set command is used, all the newly set properties will override the corresponding old properties if exists.
+     
+      Example to SET Local Dictionary Properties:
+       ```
+      ALTER TABLE tablename SET TBLPROPERTIES('LOCAL_DICTIONARY_ENABLE'='false',’LOCAL_DICTIONARY_THRESHOLD'='1000','LOCAL_DICTIONARY_INCLUDE'='column1','LOCAL_DICTIONARY_EXCLUDE'='column2')
+       ```
+      When Local Dictionary Properties are unset, default of Local Dictionary Enable will be changed to true, default of Local Dictionary Threshold will be changed to 10000, and columns for Local Dictionary Include by default will be all no-dictionary String/Varchar datatype columns.
```

Comment:
no need to repeat the default scenario, better to change it to
`When Local Dictionary properties are unset, corresponding default value will be used for those properties.`